package.json

```diff
@@ -2,19 +2,20 @@
   "name": "itowns",
   "version": "0.0.0-alpha",
   "description": "A JS/WebGL framework for 3D geospatial data visualization",
-  "main": "dist/itowns.js",
+  "main": "lib/Main.js",
   "scripts": {
     "lint": "eslint \"src/**/*.js\" \"test/**/*.js\" \"examples/**/*.js\"",
     "doc": "jsdoc src/Core/Commander/Interfaces/ApiInterface/*",
     "doclint": "npm run doc -- -t templates/silent",
     "test": "npm run lint && npm run build && npm run doclint",
     "build": "webpack -p",
     "start": "webpack-dev-server -d --inline --hot",
-    "prepublish": "npm run build"
+    "prepublish": "npm run build && BABEL_DISABLE_CACHE=1 babel src --out-dir lib"
```

Comment:
OK, let's use `npm run build` (I kind-of like having everything properly linted and tested before releasing, which is why I proposed using `npm run test`; but then we'd probably want to go the last mile of also checking that the current checkout is "clean", and properly tagged as as the appropriate version, etc. Those might be better in a "release script" than in the `prepublish` script)